src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CertificateAccess/Ressources/CertificateSchema.json

```diff
@@ -261,7 +261,8 @@
       "type": "object",
       "properties": {
         "tg": {
-          "$ref": "#/$defs/disease-agent-targeted"
+          "$ref": "#/$defs/disease-agent-targeted",
+          "minLength": 1
```

Comment:
@mlenkeit could you change this in the referenced schema in the tech-spec, so that it will be there after an update. Maybe its also something Android wants to fix.

Reply:
Will do...